code/modules/WVM/wvm.dm

```diff
@@ -679,14 +679,14 @@ GLOBAL_VAR_INIT(vendor_cash, 0)
 		new /datum/data/wasteland_equipment("Random manual",					/obj/item/book/manual/random,									40),
 		new /datum/data/wasteland_equipment("Box of ingredients - American",	/obj/item/storage/box/ingredients/american,						80),
 		new /datum/data/wasteland_equipment("Music box",						/obj/item/holodisk/musicbox,									400),
-		new /datum/data/wasteland_equipment("Advanced Armor and You",			/obj/item/book/granter/trait/pa_wear,							600),
+		new /datum/data/wasteland_equipment("Grognak the Barbarian",			/obj/item/book/granter/trait/bigleagues,						600),
```

Comment:
Adding a book to the vendor list is a little out of scope for a removal PR. I'd prefer this be added in a different PR so that they remain atomic and either can be easily reverted individually should the need arise.

Reply:
no

Reply:
I'll remove the lines myself then.

Reply:
It's such a minor thing, and forcing people to only be able to obtain Big Leagues by using the diary is silly, it'll practically force you to pick Grognak if you want to do a melee build (which also requires trekking)

Reply:
I'm not telling you that it's a bad idea, I'm telling you to make two separate PRs since these are two separate things.